lib/components/InlinePopup/InlinePopup.module.scss

```diff
@@ -25,33 +25,31 @@ $swatches-horizontal: 3;
 
         &.disabled {
             cursor: not-allowed;
-            color: var(--color-text-disabled);
+            color: var(--color-foreground-disabled);
 
             &:active, &:hover, &:focus {
                 background-color: inherit;
             }
         }
 
-        &:active, &:hover {
-            background-color: var(--color-bg-hover);
+        &:active, &:hover, &:focus {
+            background-color: var(--color-state-hover);
         }
 
         &:focus {
             outline-offset: -1px;
-            background-color: transparent;
-            outline: 1px dashed var(--color-border-focus);
+            outline: 1px dashed var(--color-accent);
         }
     }
 
     &.expanded {
         .inline-popup-label {
-            background-color: var(--color-bg-content);
-            border: 1px solid var(--color-border-rest);

```

Comment:
any reason for removing the border?

Reply:
To be consistent with what we do in renderer with popup menus, we don't ever put a border in the opening element if it didn't have one previously.

Reply:
I double checked renderer and we should add it back with `color-accent` to be consistent